plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -109,40 +109,49 @@ protected Builder()
             super(createSession(Optional.of(new SelectedRole(ROLE, Optional.of("admin")))));
         }
 
-        public Builder setSkipTimezoneSetup(boolean skipTimezoneSetup)
+        public B setSkipTimezoneSetup(boolean skipTimezoneSetup)
         {
             this.skipTimezoneSetup = skipTimezoneSetup;
-            return this;
+            return (B) this;
         }
 
-        public Builder setHiveProperties(Map<String, String> hiveProperties)
+        public B setHiveProperties(Map<String, String> hiveProperties)
         {
             this.hiveProperties = ImmutableMap.copyOf(requireNonNull(hiveProperties, "hiveProperties is null"));
-            return this;
+            return (B) this;
```

Comment:
i guess this generates a warning in the IDE, right?

would be good to have 
```
protected B self() { 
return this; 
}
```

with some proper annotations to suppress the warning